protobuf/proto/crux.proto

```diff
@@ -55,6 +55,7 @@ service CruxProductVersion {
   rpc CreateVersion(CreateVersionRequest) returns (CreateEntityResponse);
```

Comment:
Different formatting on this file again..?

Reply:
what you mean?